.eslintrc.js

```diff
@@ -6,22 +6,30 @@ module.exports = {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
     },
+
     {
       files: ['*.js'],
       parserOptions: {
         sourceType: 'script',
       },
       extends: ['@metamask/eslint-config-nodejs'],
     },
+
     {
       files: ['*.test.ts', '*.test.js'],
-      extends: ['@metamask/eslint-config-jest'],
+      extends: [
+        '@metamask/eslint-config-jest',
+        '@metamask/eslint-config-nodejs',
```

Comment:
Installed `@metamask/eslint-config-nodejs` to match module-template

design-tokens 
- https://github.com/MetaMask/design-tokens/blob/c88403b79c6547918118e5762cc2dbd386c8a14c/package.json#L49
module-template 
- https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/package.json#L49
- https://github.com/MetaMask/metamask-module-template/blob/354995b0a4040cbb74f07b42a0fbc279fb5a1ea0/.eslintrc.js#L24